app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/ChatPresenter.kt

```diff
@@ -111,11 +115,12 @@ internal class ChatPresenter(
       },
     )
     LaunchPeriodicMessagePollsEffect(
-      reportNextUntilFromPolling = { nextUntil: Instant ->
+      reportNextUntilFromPolling = { result: ChatMessagesResult ->
```

Comment:
Can we also rename this to help describe a bit better that it returns more than just the `nextUntil` after these changes?
```suggestion
      reportChatMessagesResultFromPolling = { result: ChatMessagesResult ->
```